pywbemtools/pywbemcli/_cmd_statistics.py

```diff
@@ -0,0 +1,443 @@
+cmds for managing the gathering and display of statistics that pywbemcli
+and the WBEM server can gather on the WBEM operations
+
+NOTE: Commands are ordered in help display by their order in this file.
+"""
+
+from __future__ import absolute_import, print_function
+
+import click
+
+from pywbem import Error
+
+from .pywbemcli import cli
+from ._common import warning_msg, CMD_OPTS_TXT, GENERAL_OPTS_TXT, \
+    validate_output_format, output_format_is_table, format_table, \
+    SUBCMD_HELP_TXT
+from ._common_options import add_options, help_option
+from ._click_extensions import PywbemcliGroup, PywbemcliCommand
+
+# NOTE: A number of the options use double-dash as the short form.  In those
+# cases, a third definition of the options without the double-dash defines
+# the corresponding option name, ex. 'include_qualifiers'. It should be
+# defined with underscore and not dash
+
+# Issue 224 - Exception in prompt-toolkit with python 2.7. Caused because
+# with prompt-toolkit 2 + the completer requires unicode and click_repl not
+# passing help as unicode in options as unicode
+# NOTE: Insure that all option help attributes are unicode to get around this
+#       issue
+
+
+@cli.group('statistics', cls=PywbemcliGroup, options_metavar=GENERAL_OPTS_TXT,
+           subcommand_metavar=SUBCMD_HELP_TXT)
+@add_options(help_option)
+def statistics_group():
+    """
+    Command group for WBEM server statistics.
+
+    This command group defines commands to control the display of statistical
+    data managed by pywbemcli based on the ``--timestats`` general
+    option which enables the gathering and display of statistics in pywbemcli.
+
+    Since there may also be capabilities to manage statistics in WBEM servers,
+    these commands provide for a) enabling statistics gathering in the WBEM
+    server and b) displaying the information gathered.
+
+    One component of the pywbemcli client side statistics depends on the state
+    of statistics gathering enabled in the WBEM server, reporting the server
+    response time. This appears as the column "Server Times" in the report if
+    statistics gathering in the server is enabled.
+
+    Since gathering of statistics in a WBEM server may depend on a server
+    setting, pywbemcli provides commands to enable and disable the statistics
+    gathering in the server by setting the CIM_ObjectManager
+    "GatherStatisticalData" property.
+
+    In addition to the command-specific options shown in this help text, the
+    general options (see 'pywbemcli --help') can also be specified before the
+    'statistics' keyword.
+    """
+    pass  # pylint: disable=unnecessary-pass
+
+
+@statistics_group.command('reset', cls=PywbemcliCommand,
+                          options_metavar=CMD_OPTS_TXT)
+@add_options(help_option)
+@click.pass_obj
+def statistics_reset(context):
+    """
+    Reset the counts in the client managed statistics.
+
+    This command resets the counts in the statistics kept by pywbemcli for the
```

Comment:
Change "kept by" to "gathered by", for consistency.

Reply:
DONE